Services/UI/OJS.Services.Ui.Data/Implementations/ContestsDataService.cs

```diff
@@ -51,7 +51,9 @@ public async Task<IEnumerable<TServiceModel>> GetAllPracticable<TServiceModel>()
             }
 
             return await contests
-                .OrderBy(c => c.OrderBy)
```

Comment:
Removing this order means that where its used might have changes which depend on it and this will not be valid. Please separate this logic into a different data service method and move the new ordering there. e.g. GetAllPracticeableOrderedByStartAndEndTimeDescendingAndName

Reply:
This method is only been used in GetAllAsPageByFilters() method.

Reply:
Okay